athena-aws-cmdb/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-athena-federation-sdk</artifactId>
-            <version>2021.18.1</version>
+            <version>2021.21.1</version>
```

Comment:
Would it be a good idea to update all the connector poms that have hardcoded sdk versions to be like the vertica pom that uses <aws-athena-federation-sdk.version> ?

Reply:
thats a good idea.  i'll update them

Reply:
I updated the connectors and integ test to use `<aws-athena-federation-sdk.version>`